.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
Instructs direnv to automatically load the environment defined by the Nix flake when entering the directory.

Leave the directory and direnv will unload the environment.